### DISCUSSION TITLE
希望ClashNyanpasu将来可以开发出安卓客户端。。。

### DISCUSSION THREAD

**Discussion by Meloetta-Nahida** [libnyanpasu/clash-nyanpasu#2054] [Ideas]
我前些天访问了Clash Verge官网，知道了CLashNyanpasu的存在，立刻下载试用了ClashNyanpasuV1.61，感觉效果还不错。希望ClashNyanpasu将来可以开发出安卓客户端。。。

**Comment by Slinetrac** [comment]
#324
